blocks/header/header.css

```diff
@@ -400,12 +400,36 @@ header .nav-search-panel input {
 }
 
 header .nav-search-panel .dropin-input-container {
-    width: 100%;
+  width: 100%;
 }
 
-header .nav-search-panel .search-bar__result {
-    width: 100%;
-    position: relative;
+header .nav-search-panel .search-bar-result {
+  width: 100%;
+  position: relative;
+}
+
+header .nav-search-panel .search-bar-result .product-discovery-product-list__grid {
+  margin-block: var(--spacing-small);
+}
+
+/* Popover List */
+header .nav-search-panel .search-bar-result .dropin-product-item-card:not(.dropin-product-item-card-skeleton),
+header .nav-search-panel .search-bar-result .dropin-product-item-card-skeleton > div {
+  display: grid;
+  grid-template-columns: 60px 1fr;
+}
+
+/* Product Item Skeleton */
+header .nav-search-panel .search-bar-result .dropin-product-item-card-skeleton > div > div {
+  grid-column: unset;
+}
+
+header .nav-search-panel .search-bar-result .dropin-product-item-card-skeleton > div > div .dropin-skeleton-row {
+  height: 1em;
+}
+
```

Comment:
I need to check the Product Item component in the SDK to ensure the skeleton uses the same HTML structure, so this workaround isn’t needed.